### PR TITLE
Avoid using miliseconds for retry sequence timeout

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
Related issue : APPLINK-31433

### CODE REVIEW

**Comment by LuxoftAKutsan** [comment]
@dev-gh  please review

File: src/components/policy/policy_regular/src/policy_manager_impl.cc [hunk 1]
```diff
@@ -58,7 +58,8 @@ void DeleteManager(policy::PolicyManager* pm) {
 }
 
 namespace {
-const uint32_t kDefaultRetryTimeoutInMSec = 60u * date_time::DateTime::MILLISECONDS_IN_SECOND;
+const uint32_t kDefaultRetryTimeoutInMSec =
```
Comment (dev-gh) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @LuxoftAKutsan 
This value is available from policy table, so this variable is not needed.

File: src/components/policy/policy_regular/src/policy_manager_impl.cc [hunk 2]
```diff
@@ -311,7 +312,9 @@ void PolicyManagerImpl::StartPTExchange() {
     if (update_status_manager_.IsUpdateRequired()) {
       if (RequestPTUpdate() && !timer_retry_sequence_.is_running()) {
         // Start retry sequency
-        timer_retry_sequence_.Start(NextRetryTimeout(), timer::kPeriodic);
+        const int t = NextRetryTimeout();
```
Comment (dev-gh) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @LuxoftAKutsan 
Use meaningful names for variables.

File: src/components/policy/policy_regular/src/policy_manager_impl.cc [hunk 3]
```diff
@@ -1005,10 +1008,11 @@ void PolicyManagerImpl::RetrySequence() {
   uint32_t timeout = NextRetryTimeout();
 
   if (!timeout && timer_retry_sequence_.is_running()) {
+    LOG4CXX_DEBUG(logger_, "AKutsan Stop timer " << timeout);
     timer_retry_sequence_.Stop();
     return;
   }
-
+  LOG4CXX_DEBUG(logger_, "AKutsan start " << timeout);
```
Comment (dev-gh) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @LuxoftAKutsan 
Remove this or replace with proper logging

File: src/components/policy/policy_regular/src/policy_manager_impl.cc [hunk 4]
```diff
@@ -1005,10 +1008,11 @@ void PolicyManagerImpl::RetrySequence() {
   uint32_t timeout = NextRetryTimeout();
 
   if (!timeout && timer_retry_sequence_.is_running()) {
+    LOG4CXX_DEBUG(logger_, "AKutsan Stop timer " << timeout);
```
Comment (dev-gh) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @LuxoftAKutsan 
Remove this or replace with proper logging

File: src/components/policy/policy_regular/src/policy_manager_impl.cc (hunk 3 above)
Comment (LuxoftAKutsan) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @dev-gh  107966a

File: src/components/policy/policy_regular/src/policy_manager_impl.cc (hunk 4 above)
Comment (LuxoftAKutsan) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @dev-gh  107966a

File: src/components/policy/policy_regular/src/policy_manager_impl.cc (hunk 2 above)
Comment (LuxoftAKutsan) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @dev-gh 107966a

File: src/components/policy/policy_regular/src/policy_manager_impl.cc (hunk 1 above)
Comment (LuxoftAKutsan) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @dev-gh I won't make any refactoring ( even small one) now

**Pull request merged**